skimage/filters/thresholding.py

```diff
@@ -490,3 +490,111 @@ def find_local_maxima(hist):
             return bin_centers[upper_bound]
         elif bias == 'mid':
             return bin_centers[(threshold + upper_bound) // 2]
+
+
+def threshold_mean(image):
```

Comment:
@ahojnnes see https://github.com/scikit-image/scikit-image/pull/2124#issuecomment-223073043 and https://github.com/scikit-image/scikit-image/pull/2124#issuecomment-223081464 .